clientv3/lease.go

```diff
@@ -114,6 +114,9 @@ type Lease interface {
 	Leases(ctx context.Context) (*LeaseLeasesResponse, error)
 
 	// KeepAlive keeps the given lease alive forever.
+	// If the context "ctx" is canceled or timed out,
+	// returned "LeaseKeepAliveResponse" channel is closed,
+	// and "LeaseKeepAliveResponse" from this closed channel is nil.
 	KeepAlive(ctx context.Context, id LeaseID) (<-chan *LeaseKeepAliveResponse, error)
```

Comment:
we should document about the error? for example, what happens if the client failed to send keepalive for a few minutes, and when it reconnected the lease is already expired?